frontend/src/components/pages/RegistrantExperience/index.tsx

```diff
@@ -13,10 +21,78 @@ enum RegistrantExperienceSteps {
   ReviewRegistrationPage,
 }
 
+const waiverReducer = (waiverInterface: WaiverInterface, action: any) => {
+  switch (action.type) {
+    case WaiverActions.LOADED_WAIVER: {
+      const optionalClauses: OptionalClauseResponse[] = [];
+      const requiredClauses: RequiredClauseResponse[] = [];
+      const { waiver } = action.payload;
+
+      waiver.clauses.forEach((clause: WaiverClause) => {
+        if (clause.required) optionalClauses.push({ ...clause, agreed: false });
+        else requiredClauses.push(clause);
```

Comment:
nit style: does this look cleaner as a ternary operator? `a ? b : c`

Reply:
Yep, let me change it

Reply:
Hmm ah ok, ty for checking though!